platform/security/common/src/main/java/ddf/security/ws/policy/impl/FilePolicyLoader.java

```diff
@@ -76,4 +76,8 @@ protected Document loadFromFile(URL policyFileURL) throws IOException {
   public Document getPolicy() {
     return FilePolicyLoader.policyDoc;
   }
+
+  public static void setPolicy(Document document) {
```

Comment:
❓ Shouldn't `getPolicy()` be static?

Reply:
It is implementing the interface method.